### DISCUSSION TITLE
Deta没有AUTHOR_EMAIL环境变量

### DISCUSSION THREAD

**Discussion by BoYs-J** [walinejs/waline#1843] [Q&A] [answered]
Deta环境变量里面没有找到AUTHOR_EMAIL博主邮箱的变量；

还有Deta部署后ip显示错误是解决不了的吗？看了历史问题，https://github.com/orgs/deta/discussions/282这个dis失效了

**Comment by lizheming** [comment, marked as answer]
- AUTHOR_EMAIL 已添加，更新到最新版
- deta ip 的问题是 deta 的问题，建议直接去 deta 提 issue
